tox.ini

```diff
@@ -1,14 +1,43 @@
 [tox]
 isolated_build = True
+envlist = test
+
+[testenv]
+skip_install = true
+# Requirements to run, e.g., Python tests do not include requirements for building the
+# C++ library such as cmake or conan. In particular conan freezes some package versions
+# that are also runtime requirements (pyyaml). This would be bad, since we want to
+# ensure scipp works for users, i.e., with "current" library versions that would get
+# when installing. Therefore we build the library separately. Our requirement files do
+# not include the cmake build requirements.
```

Comment:
```suggestion
# Requirements to run, e.g., Python tests do not include requirements for building the
# C++ library such as cmake or conan. In particular, conan freezes some package versions
# that are also runtime requirements (pyyaml). This would be bad, since we want to
# ensure scipp works for users, i.e., with "current" library versions that they would get
# when installing. Therefore we build the library separately.
```